grails-app/domain/org/transmartproject/db/dataquery/highdim/DeGplInfo.groovy

```diff
@@ -35,7 +35,7 @@ class DeGplInfo implements Platform {
         table         schema: 'deapp'
 
         id              column: 'platform',   generator: 'assigned'
-        genomeReleaseId column: 'release_nbr'
+        genomeReleaseId column: 'genome_build'
 
```

Comment:
@forus This pull request has not been merged yet. The two related pull request (see above) have already been merged a long time ago, leaving the current state of tranSMART being inconsistent with respect to  uploading and retrieving the genome release number. Could you help solving this issue?